src/odemis/driver/tucsen.py

```diff
@@ -0,0 +1,2530 @@
+        tuple of min, max, default, step
+
+        Raises
+        ------
+        TUCamError
+            If the actual SDK function call does not return TUCAMRET_SUCCESS
+        ValueError
+            If the requested capability does not exist.
+        """
+        #returns information about the capability, meaning its minimum, maximum, default, step.
+        if (id.value >= TUCAM_IDCAPA.TUIDC_ENDCAPABILITY.value):
+            raise ValueError("No such capability")
+        capainfo = TUCAM_CAPA_ATTR()
+        capainfo.idCapa = id.value
+        self.TUCAM_Capa_GetAttr(self.TUCAMOPEN.hIdxTUCam, pointer(capainfo))
+
+        return capainfo.nValMin, capainfo.nValMax, capainfo.nValDft, capainfo.nValStep
+
+    def set_capability_value(self, cap: TUCAM_IDCAPA, val: float) -> None:
+        """
+        Sets a capability on an open camera.
+        For a list of capabilities see the SDK,  TUCAM_IDCAPA enum
+
+        Parameters
+        ----------
+        cap : TUCAM_IDCAPA
+            The desired information identifier,
+        val : number
+            The value (float or int) to set the capability to.
+
+        Raises
+        ------
+        TUCamError
+            If the actual SDK function call does not return TUCAMRET_SUCCESS
+        ValueError
+            If the requested capability does not exist, or it is out of range
+        """
+        try:
+            capa = TUCAM_CAPA_ATTR()
+            capa.idCapa = cap.value
+            self.TUCAM_Capa_GetAttr(self.TUCAMOPEN.hIdxTUCam, pointer(capa))
+            if capa.nValMin <= val <= capa.nValMax:
+                self.TUCAM_Capa_SetValue(self.TUCAMOPEN.hIdxTUCam, capa.idCapa, val)
+            else:
+                # you asked for an out of range value
+                raise ValueError(f"Capability {cap} value {val} out of range")
+        except Exception:
+            raise ValueError("No such capability")
+
+    def get_property_info(self, id: TUCAM_IDPROP) -> Tuple[float, float, float, float]:
+        """
+        Requests property info on an open camera.
+        For a list of properties see the SDK,  TUCAM_IDCAPA enum
+        Do not use this directly, see the helper functions like _applyExposureTime
+
+        Parameters
+        ----------
+        id: TUCAM_IDPROP
+            The desired information identifier,
+
+        Returns
+        -------
+        tuple of min, max, default, step values for that property
+
+        Raises
+        ------
+        TUCamError
+            If the actual SDK function call does not return TUCAMRET_SUCCESS
+        ValueError
+            If the requested capability does not exist.
+        """
+        prop = TUCAM_PROP_ATTR()
+        prop.idProp = id.value
+        prop.nIdxChn = 0
+        try:
+            self.TUCAM_Prop_GetAttr(self.TUCAMOPEN.hIdxTUCam, pointer(prop))
+            # print('PropID=%#d Min=%#d Max=%#d Dft=%#d Step=%#d' %(prop.idProp, prop.dbValMin, prop.dbValMax, prop.dbValDft, prop.dbValStep))
```

Comment:
```suggestion
            # logging.debug('PropID=%#d Min=%#d Max=%#d Dft=%#d Step=%#d' %(prop.idProp, prop.dbValMin, prop.dbValMax, prop.dbValDft, prop.dbValStep))
```

Similarly for all other `print`